README.md

```diff
@@ -1,10 +1,10 @@
 <div align="center">
-  <h1>Saleor SDK</h1>
+  <h1>Saleor SDK - DEPRECATED</h1>
 </div>
 
 This package contains methods providing Saleor business logic for a storefront and apps. It handles Saleor GraphQL queries and mutations, manages Apollo cache, and provides an internal state to manage popular storefront use cases, like user authentication.
 
-> :warning: **Note: Saleor SDK is still under heavy development, and its API may change.**
+> :warning: **Note: Saleor SDK is being DEPRECATED. Please use authentication from our [Saleor docs](https://docs.saleor.io/docs/3.x/api-usage/authentication)**
```

Comment:
Is is being deprecated or is it deprecated already?

Reply:
good catch - fixed 🙇🏻